sql/guild_item_points.sql

```diff
@@ -23,1280 +23,1055 @@ CREATE TABLE `guild_item_points` (
-INSERT INTO `guild_item_points` VALUES ('5', '13609', '3', '1564', '4880', '2');
-INSERT INTO `guild_item_points` VALUES ('5', '13612', '5', '1440', '5040', '4');
-INSERT INTO `guild_item_points` VALUES ('5', '13621', '6', '1720', '5440', '2');
-INSERT INTO `guild_item_points` VALUES ('5', '13641', '7', '2200', '5840', '6');
-INSERT INTO `guild_item_points` VALUES ('5', '13642', '7', '3300', '5840', '6');
-INSERT INTO `guild_item_points` VALUES ('5', '13697', '1', '259', '2480', '6');
-INSERT INTO `guild_item_points` VALUES ('5', '13698', '6', '2687', '6080', '0');
-INSERT INTO `guild_item_points` VALUES ('5', '13699', '6', '2150', '5840', '6');
-INSERT INTO `guild_item_points` VALUES ('5', '13700', '6', '3440', '6400', '1');
-INSERT INTO `guild_item_points` VALUES ('5', '13701', '6', '2795', '6080', '0');
-INSERT INTO `guild_item_points` VALUES ('5', '13702', '6', '4515', '6720', '5');
-INSERT INTO `guild_item_points` VALUES ('5', '13704', '8', '4200', '6640', '1');
-INSERT INTO `guild_item_points` VALUES ('5', '13705', '8', '4560', '6720', '0');
-INSERT INTO `guild_item_points` VALUES ('5', '13706', '8', '3600', '6480', '6');
-INSERT INTO `guild_item_points` VALUES ('5', '13707', '3', '2318', '5840', '7');
-INSERT INTO `guild_item_points` VALUES ('5', '13708', '8', '5040', '6800', '4');
-INSERT INTO `guild_item_points` VALUES ('5', '13709', '4', '2072', '5600', '3');
-INSERT INTO `guild_item_points` VALUES ('5', '13739', '6', '2257', '5840', '6');
-INSERT INTO `guild_item_points` VALUES ('5', '13810', '4', '1297', '5040', '6');
-INSERT INTO `guild_item_points` VALUES ('5', '13811', '4', '1349', '5040', '6');
-INSERT INTO `guild_item_points` VALUES ('5', '13824', '2', '1140', '4640', '0');
-INSERT INTO `guild_item_points` VALUES ('5', '13835', '5', '2400', '5840', '0');
-INSERT INTO `guild_item_points` VALUES ('5', '13862', '8', '4200', '6480', '5');
-INSERT INTO `guild_item_points` VALUES ('5', '13907', '8', '5400', '6640', '1');
-INSERT INTO `guild_item_points` VALUES ('5', '13918', '9', '1080', '5040', '4');
-INSERT INTO `guild_item_points` VALUES ('5', '13918', '9', '1080', '5040', '6');
-INSERT INTO `guild_item_points` VALUES ('5', '13919', '9', '1080', '5040', '4');
-INSERT INTO `guild_item_points` VALUES ('5', '13919', '9', '1080', '5040', '6');
-INSERT INTO `guild_item_points` VALUES ('5', '13960', '6', '3547', '6400', '1');
-INSERT INTO `guild_item_points` VALUES ('5', '13992', '7', '1912', '5680', '1');
-INSERT INTO `guild_item_points` VALUES ('5', '14057', '8', '3600', '6480', '6');
-INSERT INTO `guild_item_points` VALUES ('5', '14068', '0', '63', '1440', '2');
-INSERT INTO `guild_item_points` VALUES ('5', '14069', '0', '71', '1440', '2');
-INSERT INTO `guild_item_points` VALUES ('5', '14070', '1', '360', '2880', '7');
-INSERT INTO `guild_item_points` VALUES ('5', '14071', '1', '986', '2280', '7');
-INSERT INTO `guild_item_points` VALUES ('5', '14072', '2', '519', '3520', '2');
-INSERT INTO `guild_it
```

Comment:
What value should the '/' be?

You can find these errors by looking at the two build systems (specifically travis-ci for sql changes):

```
Importing guild_item_points.sql into the database...ERROR 1064 (42000) at line 73: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '/, 800, 3)' at line 1
```